linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/LoaderContent.scala

```diff
@@ -361,15 +362,31 @@ const essentialExterns = {
   nanoTime: () => performance.now(),
   currentTimeMillis: () => Date.now(),
   random: () => Math.random(),
+
+  send: (strArray) => scalajsCom.send(wasmArrayToString(strArray)),
+  init: () => scalajsCom.init((msg) => handleMessage(msg)),
```

Comment:
I think you can use `stringToWasmArray` here:
```suggestion
  init: () => scalajsCom.init((msg) => handleMessage(stringToWasmArray(msg))),
```
That should avoid having to deal with the `stringBuiltins` in `targetPureWasm`.

Reply:
Oh thanks I didn't realize there's a `stringToWasmArray` 😄 https://github.com/scala-wasm/scala-wasm/pull/59/commits/ae4d540ec785452ed143ce6c3aff0e2a22d09ed0